docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -295,6 +307,61 @@ Multiple jobs within the same repository can target overlapping sets of assets.
 
 Like regular jobs, asset jobs can be placed on [schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors).
 
+## Testing
+
+Since a function decorated with `@asset` acts as the op for a software-defined asset, you can write unit tests for this function in the same way you would write unit tests for a standard op. See this section on [op testing](/concepts/testing#testing-ops) for some background information.
```

Comment:
Github isn't letting me reply to your earlier reply on this issue, so following up here. What I meant, but maybe didn't express clearly in my earlier comment, is that I think we should start this section in a way that allows users who come to Dagster but have never heard of the concept of an op to quickly understand what's going on.

Leading with "Since a function decorated with `@asset` acts as the op for a software-defined asset, you can write unit tests for this function in the same way you would write unit tests for a standard op." might make a user who has never heard of an op feel like they're out of their depth - they need to understand a whole new concept in order to unit test an asset.

Leading with something like "You can treat an asset definition as a regular Python function" would allow readers who have never heard of ops able to quickly grasp what's going on. I think it's still worth mentioning the op connection, just not at the top.